src/patternfly/components/OverflowMenu/docs/overflow-menu-intro.md

```diff
@@ -0,0 +1,10 @@
+The overflow menu component condenses several actions into a single dropdown button, then exposes those actions at a defined breakpoint `.pf-m-reveal{-on-[md, lg, xl]}`.
+
+The overflow menu relies on groups (`.pf-c-overflow-menu__group`) and items (`.pf-c-overflow-menu__item`), with default spacer values. Groups and items can be siblings and/or items can be nested within groups. Modifier selectors adjust spacing based on the type of group. Each modifier applies a unique CSS variable, therefore, the base spacer value for all elements can be customized and item/groups spacers can be themed individually. The default spacer value for items and groups is set to `--pf-global--spacer--md` or 16px.
```

Comment:
Do you think we should mention the default spacer value is `--pf-c-overflow-menu--spacer--base`, which is set to `--pf-global--spacer--md` or 16px?

Reply:
Or maybe represent that in the table below somehow, just to show folks they can change the global overflow-menu spacer without changing the PF global spacer var, as well as customize the item/group spacers?